.github/metaprompts/palette.md

```
@@ -0,0 +1,139 @@
+Only add entries for CRITICAL learnings:
+- An accessibility pattern specific to this app
+- A UX change that was well/poorly received
+- A design constraint to remember
+
+Format: `## YYYY-MM-DD - [Title]
+**Learning:** [UX insight]
+**Action:** [How to apply]`
+
+## Scan Checklist
+
+### Accessibility
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)